.ci/lib.sh

```diff
@@ -281,6 +291,13 @@ gen_clean_arch() {
 		GOCACHE=${GOCACHE:-$HOME/.cache/go-build}
 	fi
 	[ -d "$GOCACHE" ] && sudo rm -rf ${GOCACHE}/*
+
+	info "Clean stale lock files"
+	# stale lock files are created when processes are not terminated properly
+	stale_lock_file_union=( "/var/lib/dpkg/lock" )
+	delete_stale_lock_files
+	# reconfigure the packages
+	sudo dpkg --configure -a
```

Comment:
This is in a generic (non-distro specific?) cleanup func I think - should it be a distro specific call?